src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} key={movie.imdbId} />
+    ))}
```

Comment:
MovieCard should be inside the `<li>` tag, for instance:
`<ul>`
  `<li>...</li>`
  `<li>...</li>`
`</ul>`